packages/lesswrong/server/repos/PostsRepo.ts

```diff
@@ -8,6 +8,8 @@ export type MeanPostKarma = {
   meanKarma: number,
 }
 
+type PostAndDigestPost = DbPost & {digestPostId: string, emailDigestStatus: string, onsiteDigestStatus: string}
```

Comment:
The three digest fields are nullable in the schema, but not in this type. We should probably either make them nullable here, make them not nullable in the schema, or  add some `COALESCE`s to the query where we fetch them (especially since it's a `LEFT JOIN`).

Reply:
Ah right, I updated the type to make them nullable